.github/workflows/build.yml

```diff
@@ -5,20 +5,34 @@ on:
     ignore-branches:
       - main
       - future
+
 jobs:
   deployment:
     name: Build the image
     runs-on: ubuntu-latest
     container: docker
     steps:
       - name: Check out repository code
-        uses: actions/checkout@master
+        uses: actions/checkout@main
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: linux/arm64
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
 
       - name: Build and push the image
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
         with:
           push: false
-          tags: terraform-runner:latest
+          platforms: linux/amd64,linux/arm64
+          tags: runner-terraform:latest
+
+      # Can't use the output of the previous step because it's
+      # a multiplatform build, so we need to build it again.
+      - name: Test if terragrunt & infracost works
```

Comment:
Did you just add this for your own testing, or were you going to keep it here?

Reply:
On purpose.